web/static/js/components/surveys/SurveySimulation.jsx

```diff
@@ -212,8 +242,76 @@ class SurveySimulation extends Component {
     )
   }
 
-  render() {
+  mobileWebChatMessages() {
+    const { mode, t } = this.props
+    const { state, initialState } = this.state
+    if (mode != 'mobileweb') return []
+    const respondentIsActive = this.respondentIsActive()
+    const respondentIsPendingOrActive = state == 'pending' || respondentIsActive
+    const initialStateLoaded = this.initialStateLoaded()
+
+    let messages = []
+    if (respondentIsPendingOrActive && !initialStateLoaded) {
+      messages = [t('Waiting for messages to be sent. This could take up to 1 minute.')]
```

Comment:
These status messages are displayed as message bubbles? If so, I think that is a bit confusing.

Reply:
They aren't displayed exactly as bubbles. This is how it looks:

![Mobile web simulator (waiting)](https://user-images.githubusercontent.com/39921597/95323410-7d848f00-0874-11eb-8084-f39d9ae10fba.png)

But yes, they're similar. The styling rules are [these](https://github.com/instedd/surveda/blob/8325572a92ba0f3b5d2ed336773452905dee318a/web/static/css/_quex-simulation.scss#L227-L234).

Reply:
Display the text centered on the mobile. No bubble. Probably the text should be dark due to the "light" background. That should be enough.

Reply:
This is how it looks now:

![Mobile web simulator (waiting)](https://user-images.githubusercontent.com/39921597/95992261-25193880-0e04-11eb-9f5f-4e391e803ea9.png)